scripts/config.yaml

```diff
@@ -5,9 +5,8 @@ seasons:
   'S': [ 6, 7, 8 ]
   'F': [ 9, 10, 11 ]
 
-# only one region in this model
-regions: 
-  - 'NAmerica'
+# only one continent ('r' in osemosys, for 'region') in this model
+continent: 'NAmerica'
 
 # Provinces and states to subregions mapping
 subregions_dictionary:
```

Comment:
Shouldn't this be `regions_dictionary`? Also, if you wanted to shorten the name to just `regions` or `region_dict` or something like that, feel free to! Up to you though, just a suggestion if writing `subregions_dictionary` was getting a little tedious to write :)